protocol/src/main/java/org/apache/drill/exec/proto/UserBitShared.java

```diff
@@ -24327,11 +24336,11 @@ public Builder clearStatus() {
       "$\022\021\n\rPCAP_SUB_SCAN\020%\022\022\n\016KAFKA_SUB_SCAN\020&" +
       "\022\021\n\rKUDU_SUB_SCAN\020\'\022\013\n\007FLATTEN\020(\022\020\n\014LATE" +
       "RAL_JOIN\020)\022\n\n\006UNNEST\020*\022,\n(HIVE_DRILL_NAT" +
-      "IVE_PARQUET_ROW_GROUP_SCAN\020+*g\n\nSaslStat" +
-      "us\022\020\n\014SASL_UNKNOWN\020\000\022\016\n\nSASL_START\020\001\022\024\n\020" +
-      "SASL_IN_PROGRESS\020\002\022\020\n\014SASL_SUCCESS\020\003\022\017\n\013" +
-      "SASL_FAILED\020\004B.\n\033org.apache.drill.exec.p" +
-      "rotoB\rUserBitSharedH\001"
+      "IVE_PARQUET_ROW_GROUP_SCAN\020+\022\r\n\tJDBC_SCA" +
```

Comment:
These changes look like generated output which should not be checked in..

Reply:
This came out of compiling the protobuf, which is needed. This change looks arbitrary because the generated code in this segment appears to be ordered alphabetically.

Reply:
Got it.   Changes lgtm.  +1